irods/tasks/install_plugins.yml

```diff
@@ -7,31 +7,33 @@
       - irods-api-plugin-netcdf
     state: absent
 
-- name: Ensure netcdf rpms removed
-  ansible.builtin.file:
-    path: /root/{{ item }}-1.0-centos{{ ansible_distribution_major_version }}.rpm
-    state: absent
-  with_items:
-    - irods-icommands-netcdf
-    - irods-microservice-plugin-netcdf
-    - irods-api-plugin-netcdf
+# # XXX - netcdf is no longer accessible for iRODS 4.2.8, needs to be readded when upgrading to iRODS 4.2.11
+# - name: Ensure netcdf rpms removed
+#   ansible.builtin.file:
+#     path: /root/{{ item }}-1.0-centos{{ ansible_distribution_major_version }}.rpm
+#     state: absent
+#   with_items:
+#     - irods-icommands-netcdf
+#     - irods-microservice-plugin-netcdf
+#     - irods-api-plugin-netcdf
 
```

Comment:
This task can be removed as well. It was for removing the RPMs used to install the version 4.1 plugins removed in the previous task.